src/index.js

```diff
@@ -36,7 +36,9 @@ function addPropsTable(storyFn, context, infoOptions) {
       exclude: options.propTablesExclude,
       order: options.propTablesSortOrder,
       children: storyFn
-    }).map(c => ({ ...c })),
+    })
+      .map(c => ({ ...c }))
+      .filter(c => c && Object.keys(c).length),
```

Comment:
After the object function is 'serialized' to a plain object, stuff like fragments just become empty objects (they have no display, nor any props, so not too surprising)-- the filter catches those